src/test/java/com/google/api/generator/engine/BUILD.bazel

```diff
@@ -25,3 +27,17 @@ TESTS = [
         "@junit_junit//jar",
     ],
 ) for test_name in TESTS]
+
+TEST_CLASS_NAME = "com.google.api.generator.engine.JavaCodeGeneratorTest"
+
+# Run `bazel run src/test/java/com/google/api/generator/engine:JavaCodeGeneratorTest_update`
+# to update goldens as expected generated code.
+updated_golden(
```

Comment:
Not part of this PR, but just noticed this. Can we change this (separately) to `update_golden` as was proposed in the original doc?

Reply:
sg, was following Vadym's suggestion to make all the name of bazel target a "noun" instead of "verb". But i'm open to change it back in a separate PR.

Reply:
fixed this in PR #357